third_party/memcached_ibuf.h

```diff
@@ -0,0 +1,168 @@
+#ifndef MEMCACHED_IBUF_H_INCLUDED
```

Comment:
Please add to the commit message commit hash of small library.

Reply:
It is copied from the submodule (from its current revision in memcached). I have added this information into the commit, where I copied the files.